docs/SRS/SRS.tex

```diff
@@ -365,24 +370,182 @@ \section{Project Issues}
 
 \subsection{Open Issues}
 
+There are currently no known open issues that may lead to significant change to the product or its
+design.
+
 \subsection{Off-the-Shelf Solutions}
 
+\subsubsection{Ready-Made Products}
+
+There are existing services that solve many of the problems that this application aims to fix.
+These include AutoLeap (https://autoleap.com), Sayaaraa (https://sayaaraa.com/), and KUKUI
+(https://www.kukui.com/). These are all paid services and most offer a trial period.
+
+\subsubsection{Resuable Components}
+
+There are many libraries and frameworks available that can be reused to accelerate the building
+process of the application. Next.js can be used to provide a framework in which to build the
+application. Next.js comes with many out-of-box solutions for common website development problems.
+NextAuth.js is a library designed to help simplify the authentication process. Prisma is a library
+designed to help simplify the communication between the application and the database. Next-PWA is a
+library designed to quickly bootstrap a Next.js application into a progressive web application.
+
+All libraries and frameworks list above are free to use for private and commercial use.
+
+\subsubsection{Products That Can Be Copied}
+
+There are no known products available that can be legally copied for use in this application.
+
 \subsection{New Problems}
 
+\subsubsection{Effects on the Current Environment}
+
+This application will change the way certain processes are preformed and these changes will impact
+the users.
+
+\textbf{Work Orders}
+
+The work order system will affect the way automotive mechanics document their work. The data will
+be inputted into the application therefore any failures can result in data loss.
+
+\textbf{Appointments}
+
+The appointments system will affect the way that both the customers and the receptionists schedule
+appointments. The application will track daily appointment schedules and report time conflicts. The
+application shall not lock the receptionist out of overriding the schedule.
+
+\textbf{Quotes}
+
+The quotes system will affect the way that both the customers and automotive repair shops
+communicate quotes. The quotes will now be communicated partially or completely through the
+application instead of completely in-person. Failure in this system may lead to a loss in data.
+
+\subsubsection{Effects on the Installed Systems}
```

Comment:
Would this be relevant to add in -- that this system may replace any existing shop management systems in-use?

Reply:
> **Content**
> Specification of the interfaces between new and existing systems.
> **Motivation**
> Very rarely is a new development intended to stand completely alone. Usually the new system must coexist with some older system. This question forces you to look carefully at the existing system, examining it for potential conflicts with the new development.

I don't think so. I understood this section to be more about how will this app interact with the existing, but our app doesn't. Also the replacing in-use systems is covered in the `Effects on the Current Environment` section above.